plugins/fio_cz/src/src/Facade/UcrmFacade.php

```diff
@@ -161,4 +175,25 @@ private function sendPaymentToUcrm(array $payment): void
 
         $this->logger->info('Payment created');
     }
+
+    public function getPaymentMethod($methodName): ?string
+    {
+        if ($this->getVersion() <= 2) {
+            return "3"; // hard-coded backwards compat for 'bank transfer'
+        }
+        foreach ($this->ucrmApi->query('payment-methods') as $paymentMethod) {
```

Comment:
there will always be only bank transfer right? if so, you can probably just hard code it `4145b5f5-3bbc-45e3-8fc5-9cda970c62fb` and remove all the methodId parameters above